src/dbSta/include/db_sta/MakeDbSta.hh

```diff
@@ -47,6 +43,5 @@ class OpenRoad;
 
 sta::dbSta* makeDbSta();
 void deleteDbSta(sta::dbSta* sta);
-void initDbSta(OpenRoad* openroad);
```

Comment:
The definition remains in MakeDbSta.cc so why is the declaration moved?

Reply:
I think there were a few factors:

- Nobody relies on the public declaration, so makes sense to remove
- We can't include the public declaration into dbSta (since we're trying to break that coupling)
- If there are multiple declarations of the same thing visible in the same scope, you get a warning from `-Wredundant-decls`

Reply:
(And to be a little more explicit about the context/reasoning, note that dbSta is just exposing its guts to the function as a friend, but the friend is not defined anywhere that it can actually see. You don't need to link to something to expose your guts via `friend`, you just need to acknowledge that it's going to exist in some namespace that will mess with your guts.)

Reply:
I'm afraid I don't follow your factors.  Why can't this follow the pattern of the other modules that have been converted?